### PR TITLE
Makes Fool's Ruby use its proper scientific name

### PR DESCRIPTION
**Pull request by Alastors**
fixes: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/14443

Fools Ruby, as it is, merely serves to screw with players, literally providing no other value, sure it is a "historic quirk" of the pack, but then again, so are dense plates from ++ amongst other things, so I don't exactly see why we should keep a "historic quirk" for the sake of it.

This is a logical continuation of the precedent pushed forth by: https://github.com/GTNewHorizons/GT5-Unofficial/pull/455 in which we gave Spinel an actual use, in furthering this goal, I believe we should likewise add a vein to the ore (or even just add it to an already existing vein?) and rename the "trollstein" crop to something more appropriate.

### CODE REVIEW

**Comment by JBadges** [comment]
I really like this change. Should be merged

**Comment by Tyrantgtnh** [comment]
I really believe this would significantly improve the player experience, as a survivor of being fooled by fools ruby i will say I wouldn’t wish it on my worst enemy. Anyway, please add.

**Comment by hallucinogender** [comment]
If spinel is to be added to an ore vein, I think the method that may make the most sense (other than just creating an entirely new ore vein) would be to have spinel replace the sporadic glauconite ore in the olivine vein. This has the following advantages:
- Spinel serves as a source of magnesium, meaning it fits in well with the overall purpose of the olivine vein. It is also a minor source of aluminium, which is a function also served by glauconite.
- It is currently found in the Twilight Forest as a small ore, so the material would stay in the same place relative to overall progression.
- Reducing the availability of glauconite ore to _only_ soapstone veins shouldn't have too much impact on gameplay; everything glauconite does can also be done by various other materials.

**Comment by MatttWasTaken** [comment]
I think this would be the best addition the pack really needs at this point to really solidify everything together. Like so i've also been fooled by the foolish ruby many times myself. Definately a 10/10 PR right here. Plz add👍

File: src/main/java/gregtech/api/enums/Materials.java [hunk 1]
```diff
@@ -264,7 +264,7 @@ public class Materials implements IColorModulationContainer, ISubTagContainer {
...
-    public static Materials FoolsRuby               = new Materials( 512, TextureSet.SET_RUBY              ,   1.0F,      0,  2, 1  |4|8                   , 255, 100, 100, 127,   "FoolsRuby"               ,   "Ruby"                          ,    0,       0,         -1,    0, false,  true,   3,   1,   1, Dyes.dyeRed         , 1, Arrays.asList(new MaterialStack(Magnesium, 1), new MaterialStack(Aluminium, 2), new MaterialStack(Oxygen, 4)), Arrays.asList(new TC_AspectStack(TC_Aspects.LUCRUM, 2), new TC_AspectStack(TC_Aspects.VITREUS, 2)));
+    public static Materials FoolsRuby               = new Materials( 512, TextureSet.SET_RUBY              ,   1.0F,      0,  2, 1  |4|8                   , 255, 100, 100, 127,   "FoolsRuby"               ,   "Spinel"                          ,    0,       0,         -1,    0, false,  true,   3,   1,   1, Dyes.dyeRed         , 1, Arrays.asList(new MaterialStack(Magnesium, 1), new MaterialStack(Aluminium, 2), new MaterialStack(Oxygen, 4)), Arrays.asList(new TC_AspectStack(TC_Aspects.LUCRUM, 2), new TC_AspectStack(TC_Aspects.VITREUS, 2)));
```
Comment (Dream-Master) [inline src/main/java/gregtech/api/enums/Materials.java:267]: how the oredict looks like? did it use Spinel or False Ruby or just Ruby?

Comment (chochem) [reply src/main/java/gregtech/api/enums/Materials.java:267]: the oredict is FoolsRuby.

Comment (Alastors) [reply src/main/java/gregtech/api/enums/Materials.java:267]: > how the oredict looks like? did it use Spinel or False Ruby or just Ruby?

All I changed was the in-game name so nothing broke, it breaks a few things if you change the ID name

**Review by mitchej123 — APPROVED** [review]
I'd wait to merge in case of any other rabble rabble rabble :P

**Comment by Dream-Master** [comment]
:trollface:

**Comment by Alastors** [comment]
MergeMaster strikes again :gigachad:

**Pull request merged**